stock_picking_report_valued/report/stock_picking_report_valued.xml

```diff
@@ -1,18 +1,16 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
 
-    <template id="valued_report_picking" inherit_id="stock.report_delivery_document">
+    <template
+        id="valued_report_picking"
+        inherit_id="stock.report_delivery_document"
+        priority="17"
+    >
         <xpath expr="//div[hasclass('page')]" position="before">
             <t t-set="is_outgoing" t-value="o.picking_type_code == 'outgoing'" />
         </xpath>
-        <xpath
-            expr="//table[@t-if=&quot;o.move_line_ids and o.state==&apos;done&apos;&quot;]/thead/tr"
-            position="inside"
-        >
+        <xpath expr="//table[@name='stock_move_line_table']/thead/tr" position="inside">
             <t t-if="o.valued and o.sale_id and o.move_line_ids and is_outgoing">
-                <t t-if="o.state != 'done'">
-                    <th class="text-end"><strong>Qty Reserved</strong></th>
-                </t>
```

Comment:
i removed this because the table `stock_move_line_table` was only shown when `o.state == done`.